spec/policies/user_context_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe UserContext, [:type => :current_forwardble] do
+  let(:current_request) { Insights::API::Common::Request.new(default_request) }
+  subject do
+    described_class.new(current_request, "params")
+  end
+
+  describe "#catalog_access" do
+    let(:insights_access) { instance_double(Insights::API::Common::RBAC::Access) }
+
+    before do
+      allow(Insights::API::Common::RBAC::Access).to receive(:new).and_return(insights_access)
+      allow(insights_access).to receive(:process).and_return(insights_access)
+    end
+
+    it "fetches a memoized access list from RBAC" do
+      expect(Insights::API::Common::RBAC::Access).to receive(:new).with("").once
+      expect(insights_access).to receive(:process).once
+      subject.catalog_access
```

Comment:
@eclarizio This looks like a cut and paste error on first glance. Can we do this instead

```
2.times { subject.catalog_access }
```